### PR TITLE
#4 Create remove user from activity diagram sequence

### LINKED ISSUE #4: Create sequence diagram of remove the user from an activity

### PR DESCRIPTION
**Pull request by nikolasFreitas**
# Create remove user from activity diagram sequence

### Closes #4 

----
Diagram result:

image

### CODE REVIEW

**Comment by rodrigoprestesmachado** [comment]
We need to add a note (maybe near of DB), we can remove the user of a activity/group, but, we **can't** remove the history of a user

File: docs/sequence/remove-user-from-activity.wsd [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  actor "Activity \nparticipant" as client
+  participant BFF
+  box "Protected stack" #LightCyan
+  participant User
+  participant Activity
+  end box
+  database DB
+  note right of client #LightGreen
+    Doesn't remove the 
+    hisotry of the user in
```
Comment (GiovaniBoff) [inline docs/sequence/remove-user-from-activity.wsd:15]: Adjust this note, history is written wrong

**Review by GiovaniBoff — CHANGES_REQUESTED** [review]

**Review by evandroLutz — DISMISSED** [review]
could replace "remove me" with "remove user request"

**Comment by nikolasFreitas** [comment]
> remove user

I think the "request" is not necessary, but I will change to "remove user"

**Review by GiovaniBoff — APPROVED** [review]

**Pull request merged**